src/code.cpp

```diff
@@ -1,7 +1,27 @@
 #include <Rcpp.h>
+#include <string>
 using namespace Rcpp;
 
 // [[Rcpp::export]]
 int adder (int a, int b) {
   return(a+b);
 }
+
+// [[Rcpp::export]]
+bool is_maggie_bday (int bday) {
+  if (bday == 9251996) {
+    return(true);
+  } else {
+    return(false);
+  }
+}
+
+// [[Rcpp::export]]
+bool is_maggies_name (std::string input){
+  std::string str = "Maggie";
+  if(input == str){
+    return(true);
+  }else{
+    return(false);
+  }
```

Comment:
Bit of styling here
```suggestion
  if (input == str) {
    return(true);
  } else {
    return(false);
  }
```

Could also write this as `return(input == str)` but realise you wanted to write an if statement as an example